merlin/systems/dag/ops/operator.py

```diff
@@ -297,3 +298,27 @@ def _schema_to_dict(schema: Schema) -> dict:
         }
 
     return schema_dict
+
+
+def _add_model_param(params, paramclass, col_schema, dims=None):
+    dims = dims if dims is not None else [-1, 1]
+    if col_schema.is_list and col_schema.is_ragged:
```

Comment:
Could we update `test_tf_op_exports_own_config` to give it a model with a tupled input as well so this new logic gets covered?

Reply:
Added a new test to cover the schema inference from the model file